### ISSUE TITLE
Localization: make more strings localizable

### ISSUE THREAD

**Issue by dadler** [dadler/thumbnail-zoom#181]
Some strings embedded in .js are not currently localizable.    Areas needing localization support:
- siteConfigService.js pop-up dialog prompt "Edit disabled site URL...".
- shortcut keys, potentially (though it'd complicate documentation and support if they varied by locale).
- possibly the google images search URL (for hotkey 'g') in _doHandleKeyDown().

The help / users' manual on wordpress is English only, and supporting translation of that is out of scope of this task.

This is becoming more important as more TZP localizations are being written by translators at babelzilla.org .

---

Approach:

In some cases we've localized those via entities like &thumbnailzoomplus.save.image.title; in locale.dtd with XXXENTITYREF  in overlay.xul and getEntity() in overlay.js.  But that's hard to access from modules other than overlay.js.  We could add more XXXENTITYREF (e.g. in options.xul) and duplicate getEntity() in siteConfigService.js .  Or we could restructure those strings as follows:

We could use [properties files and stringbundles](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XUL/Tutorial/Property_Files).  The advantage is that the equivalent of getEntity() is built-in.

To do that, we'd make these changes:
1. Replace XXXENTITYREF in overlay.xul with <stringbundles> and <stringbundle>.  <stringbundle> will include a src tag referring to the properties file.  We can  add this same 4-line chunk to any xul file whose corresponding js needs properties access.  
2. Remove getEntity() from overlay.js.  Instead, callers would use code like:

``` javascript
var strbundle = document.getElementById("strings");
var nofilesfound=strbundle.getString("notFoundAlert");
```
1. Any file called from those js files can be passed the stringbundle element as an argument. 

Once that conversion is done, it'll be easier to give access to modules which can't currently call into overlay.js.
